data/data-pipeline/data_pipeline/etl/sources/maryland_ejscreen/etl.py

```diff
@@ -0,0 +1,127 @@
+        # Moreover, since the unit of measurement is at the tract level
+        # we can consistantly merge this with other datasets
+        dfs_list = [
+            df.set_index("Census_Tra").drop("geometry", axis=1)
+            for df in dfs_list
+        ]
+        # pylint: disable=unsubscriptable-object
+        combined_df = gpd.GeoDataFrame(pd.concat(dfs_list, axis=1))
+
+        # Reset index so that we no longer have the tract as our index
+        combined_df = combined_df.reset_index()
+        # coerce into integer into
+        # pylint: disable=unsupported-assignment-operation, unsubscriptable-object
+        combined_df["Census_Tra"] = (combined_df["Census_Tra"]).astype(int)
+
+        # Drop the 10 census tracts that are zero: please see here:
+        # https://github.com/usds/justice40-tool/issues/239#issuecomment-995821572
+        combined_df = combined_df[combined_df["Census_Tra"] != 0]
+
+        # Set our class instance variable after conversions in lines 50-81
+        self.df = combined_df.copy()
+
+        # Rename columns
+        self.df.rename(
+            columns={
+                "Census_Tra": self.GEOID_TRACT_FIELD_NAME,
+                "EJScore": field_names.MARYLAND_EJSCREEN_SCORE_FIELD,
+            },
+            inplace=True,
+        )
+
+        # Interpretation: An EJ score (reported as a percentile)
+        # has a percentile rank of N for some N between 0 - 100"
+        self.df[field_names.MARYLAND_EJSCREEN_PERCENTILE_FIELD] = self.df[
+            field_names.MARYLAND_EJSCREEN_SCORE_FIELD
+        ].rank(pct=True, ascending=True)
+
+        # An arbitrarily chosen threshold is used in the comparison tool output
+        self.df[field_names.MARYLAND_EJSCREEN_BURDENED_THRESHOLD_FIELD] = (
+            self.df[field_names.MARYLAND_EJSCREEN_PERCENTILE_FIELD] > 0.75
+        )
+
+        # Baseline Comparisons with some quartiles and the 90th percent OF EJ Score
+        # Interpretation: The score is greater than or equal to N% of the tracts in the state.
+        self.df[field_names.MARYLAND_EJSCREEN_TRACT_25_PERCENT_FIELD] = (
+            self.df[field_names.MARYLAND_EJSCREEN_SCORE_FIELD] >= 0.25
+        )
+        self.df[field_names.MARYLAND_EJSCREEN_TRACT_50_PERCENT_FIELD] = (
+            self.df[field_names.MARYLAND_EJSCREEN_SCORE_FIELD] >= 0.50
+        )
+        self.df[field_names.MARYLAND_EJSCREEN_TRACT_75_PERCENT_FIELD] = (
+            self.df[field_names.MARYLAND_EJSCREEN_SCORE_FIELD] >= 0.75
+        )
+        self.df[field_names.MARYLAND_EJSCREEN_TRACT_90_PERCENT_FIELD] = (
+            self.df[field_names.MARYLAND_EJSCREEN_SCORE_FIELD] >= 0.90
+        )
+
+    def load(self) -> None:
+        logger.info("Saving Maryland EJSCREEN CSV")
+        # write maryland tracts to csv
+        self.OUTPUT_CSV_PATH.mkdir(parents=True, exist_ok=True)
+        self.df[self.COLUMNS_TO_KEEP].to_csv(
+            self.OUTPUT_CSV_PATH / "maryland_ejscreen.csv", index=False
```

Comment:
Nit: all our other classes write to `"usa.csv"`, you could possibly change to that or just `"maryland.csv`", but the EJSCREEN part will be obvious through the directory.